src/Pages/Community/CommunityHome/Community.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import FilterTap from "./Components/FilterTap";
+import CommunityCardList from "./Components/CommunityCardList";
+import { API } from "../../../config";
+import "./Community.scss";
+
+class Community extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      communityCards: [],
+    };
+  }
+  componentDidMount() {
+    fetch(`${API}/data/community/data.json`)
+      .then(response => response.json())
+      .then(result => {
+        const { communityCards } = result;
+        this.setState({
+          communityCards,
+        });
+      });
+  }
+  render() {
+    return (
+      <>
+        <nav>여기에는 이후에 완성된 네비게이션 컴포넌트가 올껍니다!</nav>
```

Comment:
혹시 네비게이션 컴포넌트가 다른 컴포넌트에도 계속 사용된다면 이런식으로 Routes에서 관리하시는 걸 추천드릴게요!
```suggestion
<Router>
  <Nav />
  <Switch>
    <Route exact path="/" component={App} />
  </Switch>
  <Footer />
</Router>
```

Reply:
정확하게 이해했습니다!!
다만 모든 페이지에서 전부 사용되는 것이 아니라, 로그인이나 회원가입 페이지 같이 몇가지 페이지에서, 네비게이션이 사용되지 않기 때문에, 그런 부분을 고려해서, 저희 프로젝트에 맞게 적용하도록 해보겠습니다 😀